server/aws-lsp-codewhisperer/src/language-server/workspaceContext/workspaceFolderManager.ts

```diff
@@ -203,10 +210,13 @@ export class WorkspaceFolderManager {
 
     async clearAllWorkspaceResources() {
```

Comment:
This clearAllWorkspaceResources is in fact  a sync function, but it is marked as async